code/modules/reagents/chemistry/reagents/other_reagents.dm

```diff
@@ -733,6 +733,9 @@
 					if(H.shoes.clean_blood())
 						H.update_inv_shoes()
 			M.clean_blood()
+			M.wash_cream()
+		else if(!iscarbon(M))
```

Comment:
The else if here does not make sense, just use else

the if(iscarbon(M) earlier already checked for them being carbon, doing a second if for them not being carbon (which already know to be true) is inefficient and doesn't give you any extra benefit.